MIGRATION_STEPS.md

```diff
@@ -0,0 +1,21 @@
+## The legacy modules
+1. C# modules created from Microsoft.Azure.IoT.Edge.Module template before version 1.2.0 (inclusive).
+2. C# Edge Function modules created from Microsoft.Azure.IoT.Edge.Function before version 1.1.0 (inclusive).
+
+## Migration steps
+### Prerequisites
+Install azure-iot-edge (version 0.2.0) Visual Studio Code Extension.
+### For legacy C# modules
+1. Open the legacy module project folder in VSCode.
+2. Right click the *.csproj file and choose "Convert to IoT Edge Module".
```

Comment:
The `*` in `*.csproj file` seems to cause a Markdown rendering issue (all text afterwards become italic). You should remove it or escape it.

Reply:
`*.csproj` or \*.csproj